### DISCUSSION TITLE
Question about Maximising and Menu Bar awareness

### DISCUSSION THREAD

**Discussion by carlca** [rxhanson/Rectangle#1841] [General]
- [ ] This is a new bug that hasn't been documented in a closed issue or in the Readme.

macOS version: 14.5
Rectangle version: 0.79

Hello Ryan, I'm contacting you about this on the basis that if anyone is going to understand this issue, it'll be you!
Here is a screenshot of an email sent to the support department of Bitwig.

![SCR-20240531-ljdm-2](https://github.com/rxhanson/Rectangle/assets/260612/a40e0f5d-0897-47f0-a6cf-26aeb1add403)

This was their reply...

```
Hi Carl​,

this is not Bitwig Studio doing this, it is the default behavior of the current macOS because of displays with webcam notches.
 
Best regards,
Bitwig Technical Support Team
```

**Comment by carlca** [comment]
To which I replied "Then why has Bitwig only just started doing this since BWS 5.2b1 and why does every other macOS app on my machine behave properly?"

Am I wide of the mark here, or are Bitwig trying to wriggle out of something which is clearly down to them?

**Comment by carlca** [comment]
It turns out that even old versions of Bitwig now misbehave which does rather indicate that it's the new version of macOS 14.5 which is the problem. Yet, all other apps on my machine do not display this incorrect behaviour which, to my mind, suggests that Bitwig is misinterpreting the information that macOS is reporting about the menu bar state...

**Comment by rxhanson** [comment]
I'll have to download Bitwig and play around with it to see if I can get a better understanding of what's going on.

**Comment by carlca** [comment]
I'd appreciate that! Just don't be surprised if the ability to make music on your computer becomes a infinitely deep rabbit hole, assuming you don't indulge in such activities already 😉

**Comment by rxhanson** [comment]
I'm closing out old issues and realized I never got around to figuring this out, sorry! I'll move this over to a discussion in case anyone else wants to chime in.
